project_liberation/main_page/templates/head.haml

```diff
@@ -0,0 +1,6 @@
+%link{:rel => "stylesheet",
+		:type => "text/css",
+		:href => "https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"}
```

Comment:
Please align `rel`, `type`, `href` and others attributes which you need to add for SRI.